### DISCUSSION TITLE
Claiming of $SMR Rewards

### DISCUSSION THREAD

**Discussion by marzgithub858** [iotaledger/firefly#5159] [Help & Support]
I am one the iota stakers for the purpose of supporting the SMR and ASMB tokens. I have successfully downloaded the firefly SMR desktop v2.0.1.
My concern is that my $SMR tokens doesn't seem to show up in my balance.  I have tried opening the wallet both via seedphrase as well as thru back up file in opening the firefly shimmer desktop 2.0.1 and then "refreshing & searching" for my SMR balance to no avail.  Am I missing something here?

![SMRrewards](https://user-images.githubusercontent.com/70326803/200512337-e69ef9ac-19b8-4c56-aaf4-2a402add3517.PNG)

**Comment by PabloMosh** [comment]
The same here :(

**Reply by Buenos32** [reply]
Were you all able to get a reply for this. I was not able to migrate my SMR from Firefly wallet to Firefly SMR wallet. It was in my firefly but upon the decision to convert assembly rewards to IOTA, it totally disappeared. Do not know, what to do.

**Reply by begonaalvarezd** [reply]
Quick information about ASMB and SMR:

- Assembly: please read https://blog.iota.org/stardust-upgrade-iota-tokenomics/. The assembly project does not exist anymore, but your rewards got airdropped to you in form of IOTAs. They will appear in the new version 2.0.2 in the vesting tab. If you are missing balances you can run the balance finder from settings > advanced > balance finder, or if you are missing rewards you can also run the rewards finder from the vesting tab, the '....' menu, search vesting rewards
- Shimmer: what the old firefly version displayed was just a representation of what you owned, but you couldnt control the shimmers from the iota app. You must download the firefly shimmer app to control those funds, you can read more about it here https://blog.shimmer.network/how-to-claim-smr-firefly/

**Reply by begonaalvarezd** [reply]
Please @Buenos32  let us know if you have any issues, we released new versions of both apps this week

**Comment by Buenos32** [comment]
Dear Begonia, thanks for the message. I am getting IOTA s by vesting. Problem is that even though I have Firefly shimmer wallet, I was not able to transfer these and once new version was installed upon decision regarding assembly, my SMR staking rewards have disappeared. I am not a tech person, I have invested in IOTA thinking it was a good project.
Warm regards,
Metin Erdem

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Buenos32** [comment]
Dear Ms. De la Cruz,
I have updated both. Since I could not transfer SMR rewards to firefly smr, I have 0 funds there and no rewards at my firefly wallet.
Warm regards, have a great day!
Metin
________________________________
Gönderen: Begona Álvarez de la Cruz ***@***.***>
Gönderildi: 25 Ekim 2023 Çarşamba 16:24
Kime: iotaledger/firefly ***@***.***>
Bilgi: Buenos32 ***@***.***>; Mention ***@***.***>
Konu: Re: [iotaledger/firefly] Claiming of $SMR Rewards (Discussion #5159)


Please @Buenos32<https://github.com/Buenos32> let us know if you have any issues, we released new versions of both apps this week

**Reply by begonaalvarezd** [reply]
Hi, did you follow these steps https://blog.shimmer.network/how-to-claim-smr-firefly/ ? You need to download the latest firefly shimmer and use the option to "claim" your shimmer tokens

**Comment by Buenos32** [comment]
Dear Begona,
Thanks for the message. I was able to recover. I have opened so many wallets in order to recover this (I have like  6 now). Can I eliminate the others, if yes how?
Warm regards, have a great day!
Metin Erdem
________________________________
Gönderen: Begoña Álvarez de la Cruz ***@***.***>
Gönderildi: 27 Ekim 2023 Cuma 10:36
Kime: iotaledger/firefly ***@***.***>
Bilgi: Buenos32 ***@***.***>; Mention ***@***.***>
Konu: Re: [iotaledger/firefly] Claiming of $SMR Rewards (Discussion #5159)


Hi, did you follow these steps https://blog.shimmer.network/how-to-claim-smr-firefly/ ? You need to download the latest firefly shimmer and use the option to "claim" your shimmer tokens

**Reply by begonaalvarezd** [reply]
I you mean profiles, you can delete them from settings > profile > delete
